docs/Freestyle Tuning Principles.md

```diff
@@ -0,0 +1,285 @@
+quad behaving in a predictive and consistent manner
+
+## Betaflight Tune 
+
+### VBat Sag Compensation
+
+This feature aims to provide motor response consistency across the
+entire flight ([BF doc ref](https://github.com/betaflight/betaflight/wiki/4.2-Tuning-Notes#dynamic-battery-sag-compensation)).
+By enabling VBat Sag Compensation the craft will fly more consistently
+and predictively.
+
+If you plan to use this feature it's crucial to enable it before
+performing the PID tuning.
+
+#### Suggested setting: values for a 5"
+
+|   | **Value** |
+| ------------- | ------------- |
+| **VBat Sag Period (vbat_sag_lpf_period)**  | 200 (20 second)|
+| **VBat compensation**    | 40-60|
+
+     
+                       
+
+### PID
+
+PID are at the core of a quad tuning, with PID tuning we can achieve a
+good quadcopter attitude.
+
+D is the most important PID term to achieve smooth flying, D helps to
+minimize prop was as well as dumpening any quads movement. Freestyle
+quads tend to use higher D gains.
+
+To counterbalance a higher than usual D gain P needs to be increased as
+well.
+
+A simple approach to tune P and D is to set a desired D gain (e.g. 45)
+and slowly increase P as high as possible without producing any bounce
+back on flips and rolls (see [UAV Tech video](https://www.youtube.com/watch?v=qK5APBg76AU)).
+
+The I term is generally good enough on default, however if the quad
+feels sloppy increasing the gain could improve the overall attitude.
+
+#### Suggested settings: values for a 5"
+
+
+|   |  **P** | **I** | **D** |
+| ------------- | ------------- | ------------- | ------------- |
+| **Roll**  | 60-70  | 90-100  | 40-50  |
+| **Pitch**  | 06-70  | 90-100  | 40-50  |
+| **Yaw**  | 30-40  | 90-100  | 0  |
+
+                    
+#### Advanced Considerations
+
+**Equal setpoint tracking latency across axes**
+
+To further improve consistency it's important to properly tune all axes
+(Roll, Pitch and Yaw) this will ensure that the setpoint tracking
+latency is equal for each axe.
+
+**Motor max out**
+
+Another big factor for consistency is that when the craft is commanded
+to make a move, the motors should not max out. If that happens you'll
+get completely different responsiveness, and somewhat unpredictably.
+This is again a key factor in consistency: having enough power and
+authority on all axes.
+
+### Feed Forward
+
+Feed Forward is used to help the quad copter tracking the setpoint
+(stick inputs) closer. In other words it reduces the latency between
+stick movement and quadcopter movement.
+
+Feed Forward anticipates how the quadcopter will move based on how fast and in which direction the sticks are moved.
+
+Feed Forward is great to increase
+responsiveness but makes the craft controll less direct (more variables will affect the rotation rate) hence harder for the pilot to pridict how the quad will behave especially on RC link with low frequency frame rate (e.g. Crossfire).
```

Comment:
This is incorrect (and lots of typos). Feedforward makes control feel **more direct**. The axis rotation will track closer to the pilot's control sticks.

Reply:
1) will fix typos, sorry abut that.
2) I would prefer avoiding using the "feel" terms as it's very subjective and many freestyle pilots I talked before reducing the document disagree with this statement. So for the sake of the compromise I would like to keep this paragraph very factual.

What I am trying to convey here is that FF makes indeed the quad **more responsive** but also introduces a new "factor/thing/variable" in the quad control equation. This added "variable" can some times (especially on poor RC links e.g. crossfire) make the quad behave in a less predictable manner.

Any suggestions on how to rephrase?

Reply:
This is also false.. there is no issue if you use the right interp FF averaging to Smooth out the poor packet data from cf and other link systems (which frankly they should attempt to fix so we don’t have to bandaid it in BF ). This YouTube inspired idea that you shouldn’t run FF for freestyle flying is a general lack of understanding how it works and a bad idea to be pushing. You don’t tune FF for feel you tune it to get setpoint tracking... you using the 
Rcsmoothjng and rates to  achieve feel not FF. FF is part of the pid loop tuning to get what you ask accurately out of the quad. If does not introduce inconsistency in fact  the opposite is true. In a well tuned quad you should see, with the FF boost system the FF driving high rate moves more so than P.  I’m curious as to which ‘Devs’ you’ve been talking to...